Makefile

```diff
@@ -986,29 +987,36 @@ geojson/albers/%.geojson: geojson/%.geojson
 		| ./normalize-properties GEOID:id STATE_FIPS:id FIPS:id  \
 		> $@
 
+geojson/albers/us-10m/counties.geojson: geojson/counties.geojson
+	mkdir -p $(dir $@)
+	cat $^ \
+		| ./reproject-geojson \
+		| ./normalize-properties GEOID:id STATE_FIPS:id FIPS:id  \
+		> $@
+
```

Comment:
@anandthakker had to make this very specific to overrule an earlier make target that specified `geojson/%/counties.geojson`, which was causing an error.